### PR TITLE
Problem: tracing returnValue fix is not backported

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- backport the fix in ethermint fork and update dependency

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 # Changelog
 
-## UNRELEASED
+*March 8, 2022*
+
+## v0.6.8
```
Comment (yihuang) [inline CHANGELOG.md:5]: prepare to release a new minor version after merged.

File: go.mod [hunk 2]
```diff
@@ -164,7 +164,7 @@ replace github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethere
 // TODO: remove when ibc-go and ethermint upgrades cosmos-sdk
 replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.7-0.20220214161517-8a26cd10b4be
 
-replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.3-0.20220307013541-e3c38bb961a0
+replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.2-cronos-13
```
Comment (yihuang) [inline go.mod:167]: https://github.com/crypto-org-chain/ethermint/compare/e3c38bb961a0...v0.7.2-cronos-13?expand=1

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
 
 ### Bug Fixes
 
 - (tx, rpc) [cronos#377](https://github.com/crypto-org-chain/cronos/pull) back port bug fixes in ethermint: a) report correct gasWanted in checkTx, b) add returnValue message on tracing.
+- (rpc) [#381](https://github.com/tharsis/ethermint/issues/381) support return value on trace tx response.
```
Comment (thomas-nguy) [inline CHANGELOG.md:10]: can you fix the changelog above?

I think you removed b) so it should be only "report correct gasWanted in checkTx"

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,10 +1,13 @@
 # Changelog
 
-## UNRELEASED
+*March 8, 2022*
+
+## v0.6.8
 
 ### Bug Fixes
 
 - (tx, rpc) [cronos#377](https://github.com/crypto-org-chain/cronos/pull) back port bug fixes in ethermint: a) report correct gasWanted in checkTx, b) add returnValue message on tracing.
```
Comment (thomas-nguy) [inline CHANGELOG.md:9]:
```suggestion
- (tx, rpc) [cronos#377](https://github.com/crypto-org-chain/cronos/pull/377) back port bug fixes in ethermint: a) report correct gasWanted in checkTx, b) add returnValue message on tracing.
```

File: CHANGELOG.md (hunk 3 above)
Comment (thomas-nguy) [inline CHANGELOG.md:10]:
```suggestion
- (rpc) [#381](https://github.com/tharsis/ethermint/pull/381) support return value on trace tx response.
```

**Review by thomas-nguy — APPROVED** [review]

**Pull request merged**